app/models/user.rb

```diff
@@ -79,6 +79,19 @@ def remove_allowed_feature(feature)
     end
   end
 
+  # This is a convenience function for the common case of adding all team
+  # members, test accounts, contractors and beta testers to a new feature before
+  # general release. IMPORTANT NOTE: it assumes that only valid personnel have
+  # a @chanzuckerberg address. This method is intended to be called from the rails
+  # console.
+  def self.add_allowed_feature_for_all_czi(feature)
```

Comment:
This kind of custom actions make more sense to me as a rake task.

Reply:
true. and would that include the other `add_allowed_feature` method?

Reply:
No, what i find very specific is a the email parameter... E.g. eventually we might have a rake task for list of users, or biohub emails...